src/test/java/org/mockitousage/constructor/CreatingMocksWithConstructorTest.java

```diff
@@ -52,6 +55,12 @@ public void can_spy_abstract_classes() {
     }
 
     @Test
+    public void can_spy_abstract_classes_with_constructor_args() {
```

Comment:
Agreed - but let's first decide what the correct behavior is and then write tests for it :-)